HHS/OPRE-OPS
# Ops 1950 add user status 2

## What changed

Block login when `User.status` is not active.

## Issue

https://github.com/HHS/OPRE-OPS/issues/1950

## How to test

Set a user's status to `INACTIVE` or `LOCKED` in the DB and you should not be able to login with that user. 

## Definition of Done Checklist
